docs/resources/aws_route_tables.md.erb

```diff
@@ -8,23 +8,34 @@ Use the `aws_route_tables` InSpec audit resource to test properties of all or a
 
 <br>
 
+## Availability
+
+### Installation
+
+This resource is distributed along with InSpec itself. You can use it automatically.
+
+### Version
+
+This resource first became available in v2.1.30 of InSpec.
+
 ## Syntax
 
-  # Ensure that there is at least one route table
-  describe aws_route_tables do
-    it { should exist }
-  end
+# Ensure that there is at least one route table
+
+describe aws\_route\_tables do
```

Comment:
I think we need to add backticks or indentation here. Also we should escape the underscores.